site/reference/fn/sink/README.md

```diff
@@ -1,51 +1,45 @@
 ---
-title: "Sink"
+title: "`sink`"
 linkTitle: "sink"
 type: docs
 description: >
-   Specify a directory as an output sink package
+   Write resources to a local directory
 ---
 
 <!--mdtogo:Short
-    Specify a directory as an output sink package
+   Write resources to a local directory
 -->
 
-Implements a [sink function] by reading STDIN and writing configuration.
+`sink` reads resources from `stdin` and writes them to a local directory.
```

Comment:
If there is no directory specified, `sink` will output to stdout. Will we change this in v1?

Reply:
DIR is required argument now. This is changed in `v1`. Not sure output to `stdout` makes any sense. If that is the intention, output from `source` or `eval` should be good enough, no need of `sink`. Right ?